nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/ChartProcessor.java

```diff
@@ -1254,6 +1250,8 @@ void updateCounter() {
 
     public void setData(NMRData data, boolean clearOps) {
         setNMRData(data);
+        setDatasetType(data.getPreferredDatasetType());
```

Comment:
@brucejohnson is this the correct location to do this?

It seems to work when loading a FID, I couldn't find whether it is also called when opening a dataset (but I think it isn't).

Reply:
I don't see this getting called when opening a dataset.